### PR TITLE
Feature/azure images

### PR DESCRIPTION
**Pull request by knagasekharreddy**
According to issue # 47 below files i done to modified.

 `.scripts/bake-scalable-azure-14-0.ps1`
`.scripts/install-lansa-scalable.ps1`

I ran ./bake-scalable-azure-14-0.ps1 final execution is done.

`PS C:\gitrepos\cookbooks\scripts> ./bake-scalable-azure-14-0.ps1`

Capture done #47

Hi @VarunChopraAccolite Let me know if any thing required to do..

### CODE REVIEW

**Comment by robe070** [comment]
You need to merge into branch `feature/azure-images`

File: scripts/bake-scalable-azure-14-0.ps1 [hunk 1]
```diff
@@ -27,7 +27,7 @@ $script:IncludeDir = Split-Path -Parent $MyInvocation.MyCommand.Path
 
 Set-StrictMode -Version Latest
 
-Bake-IdeMsi -VersionText 'w12r2d-14-2-10' `
```
Comment (robe070) [inline scripts/bake-scalable-azure-14-0.ps1]: Must not check this change in

File: scripts/install-lansa-scalable.ps1 [hunk 2]
```diff
@@ -94,8 +99,21 @@ try
     Write-Host ("$(Log-Date) Shortcuts")
     #####################################################################################
 
-    copy-item "$Script:GitRepoPath\Marketplace\LANSA Scalable License\ScalableStartHere.htm" "$ENV:ProgramFiles\CloudStartHere.htm" | Out-Host
+    if ( $Cloud -eq "Azure" ) {
```
Comment (robe070) [inline scripts/install-lansa-scalable.ps1]: Unnecessary test. Remove code

File: scripts/install-lansa-scalable.ps1 [hunk 3]
```diff
@@ -94,8 +99,21 @@ try
...
+        #if ([System.IO.File]::Exists($Script:GitRepoPath\Marketplace\LANSA Scalable License\$Cloud\$VersionMajor.$VersionMinor)) {
```
Comment (robe070) [inline scripts/install-lansa-scalable.ps1]: Code as comment is not acceptable. English please

File: scripts/install-lansa-scalable.ps1 [hunk 4]
```diff
@@ -94,8 +99,21 @@ try
...
+            if (!(Test-Path -Path "$Script:GitRepoPath\Marketplace\LANSA Scalable License\$Cloud\$VersionMajor.$VersionMinor"))  {
+    #copy-item "$Script:GitRepoPath\Marketplace\LANSA Scalable License\$Cloud\$VersionMajor.$VersionMinor\ScalableStartHere.htm" "$ENV:ProgramFiles\CloudStartHere.htm" | Out-Host
```
Comment (robe070) [inline scripts/install-lansa-scalable.ps1]: Code as comment is not acceptable. English please

File: scripts/install-lansa-scalable.ps1 [hunk 5]
```diff
@@ -94,8 +99,21 @@ try
...
+    copy-item "$Script:GitRepoPath\Marketplace\LANSA Scalable License\$Cloud\ScalableStartHere.htm" "$ENV:ProgramFiles\CloudStartHere.htm" | Out-Host
+  
+  }
+     else {
+    #copy-item "$Script:GitRepoPath\Marketplace\LANSA Scalable License\$Cloud\ScalableStartHere.htm" "$ENV:ProgramFiles\CloudStartHere.htm" | Out-Host
```
Comment (robe070) [inline scripts/install-lansa-scalable.ps1]: Code as comment is not acceptable. English please

**Review by robe070 — APPROVED** [review]

**Review by robe070 — CHANGES_REQUESTED** [review]
code changes are fine. It is still merging into `master`

**Comment by VarunChopraAccolite** [comment]
hi @robe070 
thanks for your review, I will close this PR. The changes are committed.

**Pull request closed without merging**